R/S3_print.R

```diff
@@ -56,7 +56,7 @@ print_summary_lm_like <- function(x,
     fstat
   )
 
-  if (!is.null(x$proj_fstatistic)) {
+  if (exists("proj_fstatistic", x)) {
```

Comment:
maybe is.numeric here instead of exists, to be somewhat stricter and match fstat <- line above?